### PR TITLE
fix minor errors

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Fixed some minor errors.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/delete-data.md [hunk 1]
```diff
@@ -51,8 +51,8 @@ DELETE FROM {table} WHERE {filter}
 SELECT COUNT(*) FROM `rating` WHERE `rating_at` >= "2022-04-15 00:00:00" AND  `rating_at` <= "2022-04-15 00:15:00";
 ```
 
-- 若返回条数大于 1 万，请查看并使用使用[批量删除](#批量删除)。
-- 若返回条数小于 1 万，可使用此处的示例进行删除：
+- 若返回数量大于 1 万条，请参考[批量删除](#批量删除)。
+- 若返回数量小于 1 万条，可参考此处的示例进行删除：
```
Comment (shichun-0415) [inline develop/delete-data.md:55]: 看到这里用了 simple tab，示例会跟这句话挨着么？

**Review by shichun-0415 — APPROVED** [review]

**Review by yellowgomi — APPROVED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]
LGTM

**Pull request merged**